UndertaleModLib/Util/QoiConverter.cs

```diff
@@ -167,31 +188,34 @@ public unsafe static Bitmap GetImageFromSpan(ReadOnlySpan<byte> bytes, out int l
         /// Creates a QOI image as a byte array from a <see cref="Bitmap"/>.
         /// </summary>
         /// <param name="bmp">The <see cref="Bitmap"/> to create the QOI image from.</param>
+        /// <param name="disposeBMP">Whether input <see cref="Bitmap"/> should be disposed.</param>
```

Comment:
qoi _convertion_ doesn't care whether the input should be disposed or not
this should be done by the code that calls the converter, not the converter itself

Reply:
I added that for the cases like this:
Line 205 of _"UndertaleEmbeddedTexture.cs"_:
```
writer.Write(QoiConverter.GetSpanFromImage(TextureWorker.GetImageFromByteArray(TextureBlob), true,
                        writer.undertaleData.GM2022_3 ? 0 : 4));
 ```
 To dispose image that was created by `GetImageFromByteArray()`

Reply:
then the writer.Write call should be put inside a `using(Bitmap bitmap = ...GetImageFromByteArray...)`
or put only the GetSpanFromImage call in the using and store the span in a variable to make the bitmap get disposed as soon as GetSpanFromImage is done without it having to wait for Write to finish to dispose the bitmap